github/ci/prow/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -63,6 +63,38 @@ presubmits:
               - "-c"
               - "automation/check-patch.e2e-k8s.sh"
 
+    - name: pull-kubernetes-nmstate-e2e-handler-k8s-parallel
+      skip_branches:
+        - release-\d+\.\d+
+      annotations:
+        fork-per-release: "true"
+      always_run: true
```

Comment:
Let's add `skip_report: true` so results appear only at prow, when we merge the PR is merged we will set this to true and optional to false.

Reply:
Do you want to do as Daniel You and Roman suggested here ? (so it protect the case `optional` will be changed to `false`)
https://github.com/kubevirt/project-infra/pull/680/files#r513603472

Reply:
@oshoval this job is new I we don't have it at release branches there is no issue.

Reply:
Totally understand, just wonder if the problem can also happen in-place ? between the job and itself